manifests/sts-mysql.yml

```diff
@@ -2,37 +2,42 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: mysql
-  labels:
+  
     
 spec:
   replicas: 1
   selector:
     matchLabels:
+      app: app-mysql
       
   serviceName: mysql
   template:
     metadata:
       labels:
+        app: app-mysql
         
     spec:
       containers:
-      - image: 
+      - image: mysql:5.6 
         name: mysql
         env:
         - name: MYSQL_ROOT_PASSWORD
           valueFrom:
-            
+            secretKeyRef:
+              name: mysql-pass
+              key: password
         ports:
         - containerPort: 3306
           name: mysql
         volumeMounts:
-        
+        - name: mysql-data
+          mountPath: /var/lib/mysql
   volumeClaimTemplates:
   - metadata:
       name: mysql-data
     spec:
-      accessModes:
+      accessModes: ReadWriteOnce
```

Comment:
YAML is expecting array syntax here, you need to either put this on the next line and use a "-" character to make it an array like `- ReadWriteOnce,` or wrap it in quotes and brackets like `["ReadWriteOnce"]`